### PR TITLE
Support (un)registering candidate in CLI

### LINKED ISSUE #1283: CLI: add register/unregister validator support
We can vote after #1206, but we also need to be able to register/unregister candidates.

### PR DESCRIPTION
**Pull request by fyrchik**
Implement `wallet candidate (un)register`.
Move `wallet vote` to `wallet candidate vote`.

Close #1283 .

### CODE REVIEW

File: cli/wallet/validator.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+			Action:    handleVote,
+			Flags: append([]cli.Flag{
+				walletPathFlag,
+				gasFlag,
+				flags.AddressFlag{
+					Name:  "addr, a",
+					Usage: "Address to vote from",
+				},
+				cli.StringFlag{
+					Name:  "key, k",
```
Comment (roman-khimov) [inline cli/wallet/validator.go:62]: Can we make it `candidate, c`? It's a bit ambiguous to have address and key in the same command.

File: cli/wallet/validator.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+		{
+			Name:      "vote",
+			Usage:     "vote for a validator",
+			UsageText: "vote -w <path> -r <rpc> [-t <timeout>] [-g gas] -a <addr> -k <public key>",
+			Action:    handleVote,
+			Flags: append([]cli.Flag{
+				walletPathFlag,
+				gasFlag,
+				flags.AddressFlag{
+					Name:  "addr, a",
```
Comment (roman-khimov) [inline cli/wallet/validator.go:58]: Long form is long form, so it should `address, a`.

File: cli/wallet/validator.go (hunk 1 above)
Comment (roman-khimov) [reply cli/wallet/validator.go:62]: Though `wallet candidate vote ... -candidate` also doesn't look great.

File: cli/wallet/validator.go (hunk 2 above)
Comment (fyrchik) [reply cli/wallet/validator.go:58]: Fixed.

File: cli/wallet/validator.go (hunk 1 above)
Comment (fyrchik) [reply cli/wallet/validator.go]: Fixed.

File: cli/wallet/validator.go [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+				flags.AddressFlag{
+					Name:  "address, a",
+					Usage: "Address to unregister",
+				},
+			}, options.RPC...),
+		},
+		{
+			Name:      "vote",
+			Usage:     "vote for a validator",
+			UsageText: "vote -w <path> -r <rpc> [-t <timeout>] [-g gas] -a <addr> -k <public key>",
```
Comment (roman-khimov) [inline cli/wallet/validator.go:52]: `s/-k/-c`

Comment (fyrchik) [reply cli/wallet/validator.go:52]: Fixed.

File: cli/wallet/validator.go [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+	"github.com/nspcc-dev/neo-go/pkg/vm/opcode"
+	"github.com/urfave/cli"
+)
+
+func newValidatorCommands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:      "vote",
+			Usage:     "vote for a validator",
+			UsageText: "vote -w <path> -r <rpc> [-t <timeout>] [-g gas] -a <addr> -c <public key>",
```
Comment (AnnaShaleva) [inline cli/wallet/validator.go:52]: Isn't it `[-s <timeout>]` (not `-t`)? [link](https://github.com/nspcc-dev/neo-go/blob/master/cli/options/options.go#L34)

Comment (fyrchik) [reply cli/wallet/validator.go:52]: Fixed.

**Pull request merged**